drivers/sensor/adxl345/adxl345.c

```diff
@@ -17,18 +17,115 @@
 
 LOG_MODULE_REGISTER(ADXL345, CONFIG_SENSOR_LOG_LEVEL);
 
+#define ADXL345_BUS_I2C DT_ANY_INST_ON_BUS_STATUS_OKAY(i2c)
```

Comment:
i2c/spi includes should be conditional (spi one is missing actually), see e.g.

https://github.com/zephyrproject-rtos/zephyr/blob/ec2b58c01cdc9adc177556d0807a2881249d9273/drivers/sensor/iis2dlpc/iis2dlpc.c#L19-L23

Reply:
Both SPI and I2C may exist at the same time.
Checking separately not using if ~ else.
Also, I fixed it to #include <zephyr / ...> as a whole.